### DISCUSSION TITLE
[Question][Hw4] Question about coverage evaluation.

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#577] [Q&A]
Is the coverage evaluated by

1. From all the generated inputs, the maximum coverage(from a single input)

2. From all the generated inputs, the union of all the coverage

I am sorry for keep asking similar questions...

**Comment by RiceBiscuits** [comment]
We will union all the coverage.

**Comment by ee12ha0220** [comment]
Thanks! I was thinking it was like (1)....
